app/src/main/java/com/example/myapplication/model/retrofit/RetrofitClient.kt

```diff
@@ -5,17 +5,17 @@ import retrofit2.Retrofit
 import retrofit2.converter.gson.GsonConverterFactory
 import retrofit2.http.GET
 
-class RetrofitClient {
-    val BASE_URL = "https://api.nasa.gov/"
+
+class RetrofitClient{
+    val BASE_URL ="https://api.nasa.gov/"
```

Comment:
might be worth extracting this api to its own val

`val API_KEY =W1mGbLHETYWSrag3piW2i5PxoQxmaxv0wPC9Vg8E&feedtype=json&ver=1.0`